chia/types/eligible_coin_spends.py

```diff
@@ -353,13 +357,6 @@ async def process_fast_forward_spends(
 
         # Update bundle_coin_spends using the collected data
```

Comment:
"collected data" is very vague. it could mean anything. I think it refers to the new coin spends (i.e. "rebased" singleton spends) and it used to mean the `SPendBundleConditions`.

Reply:
It didn't use to mean `SpendBundleConditions`, it always meant the map of rebased singleton coin ID to coin spend and metadata, hence the name `ff_bundle_coin_spends`. I've added a comment accordingly.